.travis.yml

```diff
@@ -3,7 +3,7 @@ matrix:
     - os: osx
       osx_image: xcode9.0
       language: node_js
-      node_js: "6"
+      node_js: "8"
```

Comment:
Just to confirm.  Once this is moved in we will need to use NodeJS 8?  Will earlier versions work?  Not an issue just want to know.